src/semeio/forward_models/__init__.py

```diff
@@ -0,0 +1,135 @@
+from ert.config import ForwardModelStepJSON, ForwardModelStepPlugin
```

Comment:
These two symbols should be exposed through `ert/__init__.py` so that we can import it like this:
```
from ert import ForwardModelStepJSON, ForwardModelStepPlugin
```

Reply:
This way we hide detail about internal structure in ert.